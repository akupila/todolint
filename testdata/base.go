package base

// OK

// TODO(test): Make a pizza
// FIXME(test): Thinner crust
// BUG(test): Remove pineapple
/*
TODO: Multi-line comments are ignored
*/

// Error

// This is a TODO          // want `TODO should be at the beginning of the line`
// TODO(test)Make a pizza  // want `TODO should follow the format '// TODO\(context\): text'`
// TODO(test) Make a pizza // want `TODO should follow the format '// TODO\(context\): text'`
// TODO(test):Make a pizza // want `TODO should follow the format '// TODO\(context\): text'`
// TODO                    // want `TODO should include additional context: TODO\(<context>\)`
// TODO(): Make a pizza    // want `TODO should include additional context: TODO\(<context>\)`
// TODO: Make a pizza      // want `TODO should include additional context: TODO\(<context>\)`
// TODO(test):             // want `TODO should describe what needs to change`
// TODO(~): Make a pizza   // want `TODO context does not match regular expression: \\w+`
