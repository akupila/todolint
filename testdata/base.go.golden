package base

// OK

// TODO(test): Make a pizza
// FIXME(test): Thinner crust
// BUG(test): Remove pineapple
/*
TODO: Multi-line comments are ignored
*/

// Error

// This is a TODO          // want `TODO should be at the beginning of the line`
// TODO(test): Make a pizza
// TODO(test): Make a pizza
// TODO(test): Make a pizza
// TODO                    // want `TODO should include additional context: TODO\(<context>\)`
// TODO(): Make a pizza    // want `TODO context does not match regular expression: \\w+`
// TODO: Make a pizza      // want `TODO should include additional context: TODO\(<context>\)`
// TODO(test):             // want `TODO should describe what needs to change`
